worker/wallet.js

```diff
@@ -118,3 +119,20 @@ export function checkWithdrawal ({ boss, models, lnd }) {
     }
   }
 }
+
+export function autoDropWdInvoices ({ models }) {
+  return async function () {
+    console.log('forgetting invoices')
+    try {
+      await serialize(models, models.$executeRaw`
+      UPDATE "Withdrawl" SET
+        hash = '${FORGOTTEN_HASH}',
+        bolt11 = '${FORGOTTEN_HASH}'
```

Comment:
Is this tested? I don't think this will work since `Invoice.hash` has an unique index:

```
stackernews=# \d "Invoice.hash_unique"
Index "public.Invoice.hash_unique"
 Column | Type | Key? | Definition
--------+------+------+------------
 hash   | text | yes  | hash
unique, btree, for table "public.Invoice"
```

Reply:
Interesting. I certainly had multiple invoices in my testing...

Reply:
You are definitely correct that this causes errors. Thinking about the database implementation more, I think the right way to implement this is to null the fields instead of putting in a special string. The reality is that those fields are no longer "required" to be valid, so allowing null would make sense IMO. Let me know if you disagree.